Source/WebKit/UIProcess/API/glib/WebKitWebView.cpp

```diff
@@ -287,6 +287,7 @@ struct _WebKitWebViewPrivate {
     HashSet<unsigned> frameDisplayedCallbacksToRemove;
 #endif
 
+    RefPtr<API::PageConfiguration> configurationForNextRelatedView;
```

Comment:
This is weird, but I don't think there's a better way to do it. Maybe we can find a better name.

Reply:
It's the best name I can think of.